chronicle-front/src/app/components/ticket-approval/ticket-approval.component.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, OnInit } from '@angular/core';
+import { Ticket } from 'src/app/models/Ticket';
+import { TicketService } from 'src/app/services/ticket.service';
+
+@Component({
+  selector: 'app-ticket-approval',
+  templateUrl: './ticket-approval.component.html',
+  styleUrls: ['./ticket-approval.component.css']
+})
+export class TicketApprovalComponent implements OnInit {
+
+  underReviewTickets:Ticket[] =[]
+  allSubmittedTickets:Ticket[]=[]
+  displayWaitingForReview:boolean = true;
+  waitingForReview_nav_color = "orange";
+  inProgress_nav_color = "grey";
+
+
+  tempTicket:Ticket = new Ticket(0,'0','0',new Date(),new Date(),"ticket", "", "", "","", "", 0, "", "", "","");
+  rejectComment:string = "ghgh";
```

Comment:
What is the significance of the default value of this String "ghgh"?

Reply:
We set it empty string.